### PR TITLE
98: Change any unit tests not conforming to the new unit test naming convention

### PR DESCRIPTION
**Pull request by shanek21**
This change is [Reviewable](https://reviewable.io/reviews/chauncy-crib/tagprobot/100)

### CODE REVIEW

File: test/asserts.spec.js [hunk 1]
```diff
@@ -2,24 +2,36 @@ import test from 'tape';
 import { assert, assertGridInBounds } from '../src/utils/asserts';
 
 
-test('test assert', t => {
-  let condition = false;
+test('assert: does not throw errors with condition=true', t => {
+  const condition = true;
+  t.doesNotThrow(() => { assert(condition); });
+
```
Comment (davidabrahams) [inline test/asserts.spec.js]: i think i like whitespace less than you do

Comment (shanek21) [inline test/asserts.spec.js]: Is that a suggestion to take away the newline before every `t.end();`?

Comment (davidabrahams) [inline test/asserts.spec.js]: haha yeah, for short tests like this at least

Comment (shanek21) [inline test/asserts.spec.js]: ...but :white_large_square: :space_invader:

Comment (davidabrahams) [inline test/asserts.spec.js]: haha it's fine

**Comment by davidabrahams** [comment]
Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/chauncy-crib/tagprobot/100)*

**Comment by davidabrahams** [comment]
Reviewed 1 of 2 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/chauncy-crib/tagprobot/100)*

**Comment by davidabrahams** [comment]
:dango:

---

Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/chauncy-crib/tagprobot/100#-:-Kqz86cIqBbh0ql2YWTg:bcbqss6)*

**Pull request merged**